pvlib/tests/test_solarposition.py

```diff
@@ -770,14 +771,14 @@ def test__datetime_to_unixtime_units(unit, tz):
 
 
 @requires_pandas_2_0
+@pytest.mark.parametrize('tz', [None, 'utc', 'US/Eastern'])
 @pytest.mark.parametrize('method', [
     'nrel_numpy',
     'ephemeris',
     pytest.param('pyephem', marks=requires_ephem),
     pytest.param('nrel_numba', marks=requires_numba),
     pytest.param('nrel_c', marks=requires_spa_c),
 ])
-@pytest.mark.parametrize('tz', [None, 'utc', 'US/Eastern'])
```

Comment:
This order change was so that all the numba tests run in sequence; the previous ordering interleaved numpy & numba calls, causing unnecessary reloading of `pvlib.spa`.